src/javascript/app_2/App/Components/Elements/PositionsDrawer/helpers/positions-helper.js

```diff
@@ -0,0 +1,43 @@
+import moment       from 'moment';
+import { localize } from '_common/localize';
+
+export const addCommaToNumber = (number) => number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+
+export const getTimePercentage = (start_time, purchase_time, expiry_time) => {
+    const duration_from_purchase = moment.duration(moment.unix(expiry_time).diff(moment.unix(purchase_time)));
+    const duration_from_now = moment.duration(moment.unix(expiry_time).diff(start_time));
```

Comment:
does start_time mean now? I'm a bit confused about how this gives us duration from now